README.md

```diff
@@ -9,12 +9,12 @@ For additional Plotters, see the [Community Plotters](https://github.com/gonum/p
 
 There is a discussion list on Google Groups: gonum-dev@googlegroups.com.
 
-`gonum/plot` is split into a few packages:
+`gonum/plot` is split into following packages:
```

Comment:
`s/following/the following/`

Reply:
hmm... I don't see this is a necessary improvement.

Reply:
"the following"

Reply:
English grammar demands the definite article here before "following".